src/SFA.DAS.EmployerDemand.Domain/Demand/Api/Responses/GetProviderEmployerDemandResponse.cs

```diff
@@ -27,4 +29,9 @@ public class ProviderEmployerDemandItem
         [JsonProperty("employersCount")]
         public int Employers { get; set; }
     }
+    public class Sector
```

Comment:
same comment with this - if a route and sector are semantically the same then how can a sector have a child route?? best renaming class Sector to Route and property Route to RouteName or the like.